tools/Azure.Mcp.Tools.Sql/src/Commands/Database/DatabaseCreateCommand.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Extensions;
+using Azure.Mcp.Core.Models.Command;
+using Azure.Mcp.Tools.Sql.Commands;
+using Azure.Mcp.Tools.Sql.Models;
+using Azure.Mcp.Tools.Sql.Options;
+using Azure.Mcp.Tools.Sql.Options.Database;
+using Azure.Mcp.Tools.Sql.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.Sql.Commands.Database;
+
+public sealed class DatabaseCreateCommand(ILogger<DatabaseCreateCommand> logger)
+    : BaseDatabaseCommand<DatabaseCreateOptions>(logger)
+{
+    private const string CommandTitle = "Create SQL Database";
+
+    private readonly Option<string> _skuNameOption = SqlOptionDefinitions.SkuNameOption;
+    private readonly Option<string> _skuTierOption = SqlOptionDefinitions.SkuTierOption;
+    private readonly Option<int> _skuCapacityOption = SqlOptionDefinitions.SkuCapacityOption;
+    private readonly Option<string> _collationOption = SqlOptionDefinitions.CollationOption;
+    private readonly Option<long> _maxSizeBytesOption = SqlOptionDefinitions.MaxSizeBytesOption;
+    private readonly Option<string> _elasticPoolNameOption = SqlOptionDefinitions.ElasticPoolNameOption;
+    private readonly Option<bool> _zoneRedundantOption = SqlOptionDefinitions.ZoneRedundantOption;
+    private readonly Option<string> _readScaleOption = SqlOptionDefinitions.ReadScaleOption;
+
+    public override string Name => "create";
+
+    public override string Description =>
+        """
+        Create a new Azure SQL Database on an existing SQL Server. This command creates a database with configurable
+        performance tiers, size limits, and other settings. Equivalent to 'az sql db create'.
+        Returns the newly created database information including configuration details.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new()
+    {
+        Destructive = false,
+        Idempotent = true,
```

Comment:
Should idempotent here be false? Shouldn't calling create database multiple times with the same input error in subsequent calls with a 409 or something?

Reply:
addressed